### PR TITLE
Update on Tor Browser

### LINKED ISSUE #177: Tor Browser route update is required due to major release
See https://blog.torproject.org/new-release-tor-browser-115/

The route needs to be updated with updated/new information

### PR DESCRIPTION
**Pull request by pterocles**
Tor Browser Bundle (TBB) now has easier settings for the user. Only the desktop version of Tor Browser offers automatic bridges through Moat. Tor is making many strides in both usability and configuration, as well as functionality.

TBB also makes the connection simpler for the user:
- Now enabled by default, moat will help you configure bridges.
- Bridge cards for viewing your Tor bridges make it both pretty and more useful for the user.

Fixes #177

### CODE REVIEW

**Comment by NobodySpecial256** [comment]
https://github.com/Anon-Planet/thgtoa/blob/master/CONTRIBUTING.md

> Do make sure the title of a draft PR makes it immediately clear that it's a draft

**Comment by pterocles** [comment]
I didn't see you viewing this PR for...a while, so I didn't mark it yet.

File: guide.md [hunk 1]
```diff
@@ -551,7 +551,7 @@ Here are some online resources you can use to find some information about your c
 
     -   <https://check.torproject.org>
 
-For those reasons, you will need to obfuscate and hide that origin IP (the one tied to your identification) or hide it as much as we can through a combination of various means:
+For those reasons, you will need to obfuscate and hide that origin IP (the one tied to your identification) or hide it through a combination of various means:
```
[the remark this answers is not in the document]
Comment (pterocles) [reply guide.md:554]: Tor completely hides your original IP from the destination. It's a certainty. The lines after provide the means to do this.

File: guide.md [hunk 2]
```diff
@@ -579,9 +579,9 @@ Because it is not encrypted, your ISP and/or any other adversary could still int
...
+A solution to this is to use encrypted DNS using DoH (DNS over HTTPS[^36]), DoT (DNS over TLS[^37]) with a private DNS server (this can be self-hosted locally with a solution like pi-hole[^38], remotely hosted with a solution like nextdns.io or using the solutions provided by your VPN provider or the Tor network). This should prevent your ISP or some go-between from snooping on your requests ... except it might not.
 
-Small in-between Disclaimer: **This guide does not necessarily endorse or recommends Cloudflare services even if it is mentioned several times in this section for technical understanding.**
+Small in-between Disclaimer: **This guide does not necessarily endorse or recommend Cloudflare services even if it is mentioned several times in this section for technical understanding.**
```
[the remark this answers is not in the document]
Comment (pterocles) [reply guide.md:584]: No, it's not, the guide is singular:

Guide is the subject: "This guide" 
So "does not necessarily recommends" is grammatically wrong.

If it were a plural subject, the word would be "recommends".

File: guide.md [hunk 3]
```diff
@@ -2038,6 +2043,10 @@ Please see [Appendix Y: Installing and using desktop Tor Browser].
 
 ### Android:
 
+**Note on Tor Browser for Android: The development of Tor Browser for Android is behind desktop Tor Browser Bundle (TBB). Some features are not available yet. E.g., the desktop version of Tor now enables automatic bridges using Moat:**
```
[the remark this answers is not in the document]
Comment (pterocles) [reply guide.md:2046]: The notice should be placed in relevant areas like Android and iOS as well as the Desktop Tor setup.

File: guide.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (pterocles) [reply guide.md:584]: I'm not an English teacher but I'm fluent in the language. It's "recommend".

[the remark this answers is not in the document]
Comment (pterocles) [reply guide.md:584]: No, I'm not "just wrong" and respect me by not calling me dude. It's literally "This guide does not recommend".

**Pull request merged**